.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: true
```

Comment:
Yes I needed sudo for some moment in time ( external scripts required it for some reason ) now I'll switch it back and test if it works without sudo. Good catch, thanks.